plugin/storage/kafka/options.go

```diff
@@ -74,15 +83,37 @@ func (opt *Options) AddFlags(flagSet *flag.FlagSet) {
 		defaultEncoding,
 		fmt.Sprintf(`Encoding of spans ("%s" or "%s") sent to kafka.`, EncodingJSON, EncodingProto),
 	)
+	flagSet.Int(
```

Comment:
can this be a string field with readable values? e.g. noack, local, all

Reply:
fixed